audit.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
We'll need to include this in Docker build so auditing works in Docker build as well.

cc @cpanato

Reply:
Cloud typically configures this via ENV vars instead of file, so I don't think we need to ship the sample config.

Reply:
We need to include something so users can run Focalboard from Dockerfile. Without this file, the server won't start.

Reply:
Never mind, I see we updating audit initialization. So we're good here.